### PR TITLE
Using Loguru instead of prints

### PR DESCRIPTION
**Pull request by mmphego**
This PR addresses this:
- Used `Black` for reformatting.
- Using [loguru](https://github.com/Delgan/loguru) module for handling all the logging related issues.
- Better exception handling.

# Note: This is code is untested.

### CODE REVIEW

**Comment by mmphego** [comment]
@imdhanush see changes I made.

File: python_examples/sinric/_cbhandler.py [hunk 1]
```diff
@@ -1,80 +1,94 @@
...
+        if jsn.get("action") == "setPowerState":
             try:
-                power_callback = self.callbacks.get('setPowerState')
-                power_callback(jsn.get('deviceId'), jsn['value'])
-            except Exception as e:
-                logging.info(e)
+                power_callback = self.callbacks.get("setPowerState")
+                power_callback(jsn.get("deviceId"), jsn["value"])
+            except Exception:
+                self.logger.exception("Error Occurred")
```
Comment (imdhanush) [inline python_examples/sinric/_cbhandler.py]: self.logger not defined in _cbhandler.py

File: python_examples/sinric/_sinricsocket.py [hunk 2]
```diff
@@ -1,37 +1,45 @@
...
+logger.add("{}.log".format(__name__), rotation="10 MB")
 
 
 class SinricSocket:
     def __init__(self, apikey, callbacks):
-        self.apikey = apikey.encode('ascii')
+        self.apikey = apikey.encode("ascii")
         self.callbacks = callbacks
+        self.logger = logger
         self.callback_handler = CallBackHandler(self.callbacks)
```
Comment (imdhanush) [inline python_examples/sinric/_sinricsocket.py]: pass self.logger as parameter next to self.callbacks. So you can use self.logger in _cbhandler.py

**Comment by mmphego** [comment]
@imdhanush Updated.

**Comment by mmphego** [comment]
@kakopappa Please review and merge.

**Comment by kakopappa** [comment]
Thanks guys

**Pull request merged**